examples/decoding/plot_mixed_gambles_space_net.py

```diff
@@ -0,0 +1,46 @@
+"""
+SpaceNet on Jimura et al "mixed gambles" dataset.
+==================================================
+
+The segmenting power of SpaceNet is quite visible here.
+"""
+# author: DOHMATOB Elvis Dopgima,
+#         GRAMFORT Alexandre
+
+
+### Load data ################################################################
+import numpy as np
+import nibabel
+from scipy import ndimage
+from nilearn.datasets import fetch_mixed_gambles
+data = fetch_mixed_gambles(n_subjects=16, return_raw_data=True)
```

Comment:
You understood the semantics of my suggestion the opposite way. For me, return_raw_data is not what you are doing here, but the opposite (it's just a change from True to False).